### PR TITLE
Adds an extra ghost warning to defibrilations when pads start setting up

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[tweak]

## What this does
adds this if ghosts can be reasonably revived when defibrillator pads are applied, with all the relative checks except the clothing covering ones due to probability.
also some code cleanup with the ghost alert message along the way to cut down on repetition of it.

## Why it's good
notifies observers in more due time that they're about to come back this way.

## Changelog
:cl:
 * tweak: Ghosts are now aware of being revived via defibrillators sooner.
 * tweak: Ghosts now know specifically if defibrillators are bringing them back.

### CODE REVIEW

File: code/game/objects/items/weapons/defib.dm [hunk 1]
```diff
@@ -130,6 +130,8 @@
 /obj/item/weapon/melee/defibrillator/proc/attemptDefib(mob/living/carbon/human/target,mob/user)
 	user.visible_message("<span class='notice'>[user] starts setting up the paddles on [target]'s chest.</span>", \
 	"<span class='notice'>You start setting up the paddles on [target]'s chest</span>")
+	if(target.mind && !target.client && target.get_heart() && target.get_organ(LIMB_HEAD) && target.has_brain() && !target.mind.suiciding && target.health+target.getOxyLoss() > config.health_threshold_dead)
+		target.ghost_reenter_alert("Someone is about to try to defibrilate your body. Return to it if you want to be resurrected!")
```
Comment (SonixApache) [inline code/game/objects/items/weapons/defib.dm]: `target.ghost_reenter_alert("Someone is about to try to defibrillate your body. Return to it if you want to be resurrected!")`

File: code/game/objects/items/weapons/defib.dm [hunk 2]
```diff
@@ -163,17 +165,7 @@
...
-				var/mob/ghostmob = ghost.get_top_transmogrification()
-				if(ghostmob)
-					ghostmob << 'sound/effects/adminhelp.ogg'
-					to_chat(ghostmob, "<span class='interface big'><span class='bold'>Someone is trying to revive your body. Return to it if you want to be resurrected!</span> \
-						(Verbs -> Ghost -> Re-enter corpse, or <a href='?src=\ref[ghost];reentercorpse=1'>click here!</a>)</span>")
-					target.visible_message("<span class='warning'>[src] buzzes: Defibrillation failed. Vital signs are too weak, please try again in five seconds.</span>")
-					return
-			//we couldn't find a suitable ghost.
-			target.visible_message("<span class='warning'>[src] buzzes: Defibrillation failed. No brainwaves detected.</span>")
+			target.visible_message("<span class='warning'>[src] buzzes: Defibrillation failed. [target.ghost_reenter_alert("Someone has tried to defibrilate your body. Return to it if you want to be resurrected!") ? "Vital signs are too weak, please try again in five seconds" : "No brainwaves detected"].</span>")
```
Comment (SonixApache) [inline code/game/objects/items/weapons/defib.dm]: `target.visible_message("<span class='warning'>[src] buzzes: Defibrillation failed. [target.ghost_reenter_alert("Someone has tried to defibrillate your body. Return to it if you want to be resurrected!") ? "Vital signs are too weak, please try again in five seconds" : "No brainwaves detected"].</span>")`

**Comment by SonixApache** [comment]
good stuff, fix the typos
do want to note that this will make spam-clicking the defib even *more* obnoxious

File: code/game/objects/items/weapons/defib.dm (hunk 1 above)
Comment (SECBATON-GRIFFON) [reply code/game/objects/items/weapons/defib.dm]: fixed typo

File: code/game/objects/items/weapons/defib.dm (hunk 2 above)
Comment (SECBATON-GRIFFON) [reply code/game/objects/items/weapons/defib.dm]: fixed typo

**Pull request merged**